### DISCUSSION TITLE
Can AC Stark Map calculate lightshift by sigma+/sigma- transitions?

### DISCUSSION THREAD

**Discussion by hankwqw** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#184] [Q&A] [answered]
I tried to calculate the lightshift on rubidium 87 D1 and D2 lines with AC Stark Map. I choose pi polarization and it works pretty well. While I choose sigma+/sigma- transitions, it comes out zero at all of these wavelengths. I use ShirlyMethods and find the calculation above. I'm wondering what's wrong with it.

**Comment by dihm** [comment, marked as answer]
Are you using the latest release (v3.6.0)? There were fixes to the AC Stark Map functionality specifically relating to using circular polarizations.

Also, I'm not entirely sure the AC Stark Map will work correctly for optical wavelengths. On the one hand, I don't believe there are assumptions that preclude optical transitions. On the other hand, it is not what the code was designed for and I have not preformed any tests there. It is possible you might be better served using the `DynamicPolarisability` of ARC instead. Example of its usage to find magic trap wavelengths are in the [docs](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/ARC_3_0_introduction.html#Calculations-of-dynamic-polarisability-and-magic-wavelengths-for-optical-traps).
